pages/search/index.tsx

```diff
@@ -20,33 +22,38 @@ export default function Search({ results }) {
   const { query } = useRouter()
   const searchParams = mapQueryToSearchParams(query)
 
+  const { itemListElement, totalResults } = results.results
+
+  const searchResultBibs = itemListElement.map((result: SearchResultsItem) => {
```

Comment:
Try this: manually make `itemListElement=[{}, {}, {}]` or any variation with undefined or null. Because this is a `map` and is returning `null` for the first case, the `searchResultsBibs` variable can turn out to be `[undefined, { ...}, { ... }, undefined, ...]` if the returned results from the API are wrong (empty or null or undefined).

In this case a forEach might be better.

Reply:
Good catch, I moved the null check into a filter before the map.